app/views/media_library/show.html.erb

```diff
@@ -1 +1,81 @@
-<p>Hello from media library show</p>
+<div class="l-header-single -media-library">
+  <div class="overlay"></div>
+  <%= render 'shared/menu' %>
+  <div class="row">
+    <div class="small-12 columns">
+      <div class="title-block">
+        <div class="c-breadcrumbs">
+          <a href="/">
+            <svg class="icon-home">
+              <use xlink:href="#icon-breadcrumb-home"></use>
+            </svg>
+          </a>
+          <svg class="icon">
+            <use xlink:href="#icon-breadcrumb-chevron-right"></use>
+          </svg>
+          <%= link_to "Media library", media_path, class: "text -meta" %>
+          <svg class="icon">
+            <use xlink:href="#icon-breadcrumb-chevron-right"></use>
+          </svg>
+          <span class="text -meta">Arctic Biodiversity</span>
+        </div>
+        <div class="share">
+          <svg class="icon">
+            <use xlink:href="#icon-share"></use>
+          </svg>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="l-media-article -media-library">
+
+  <div class="row">
+    <div class="small-12 column">
+      <div class="c-article-image"></div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="small-12 column">
+      <div class="content">
+        <h2 class="text -title-single">Polar bear (Ursus maritimus), Northwest Spitsbergen National Park, Svalbard</h2>
+        <div class="text -body-single">
+          <p>Svalbard is the best example of Linking Tourism & Conservation and where totally protected polar bears can be observed under well-regulated and -managed conditions.</p>
+          <br>
+          <p><strong>Year:</strong> 2014</p>
+          <p><strong>From collection:</strong> Arctic Biodiversity</p>
+          <p><strong>Taken by:</strong> Peter Prokosch</p>
+          <p><strong>Downloads:</strong> 4509 <span class="rating"><strong>Rating:</strong></span></p>
+          <p class="tags">
+            <strong>Tags:</strong>
+            <span class="tag text -meta">Arctic</span>
+            <span class="tag text -meta">Biodiversity</span>
+            <span class="tag text -meta">Conservation</span>
+          </p>
+        </div>
+      </div>
+
+      <div class="sidebar">
```

Comment:
Content and sidebar could be split into two different grid divs, so we don't need to set up rules for responsive behavior.